test/e2e/framework/service_util.go

```diff
@@ -94,6 +94,10 @@ const (
 	// GCPMaxInstancesInInstanceGroup is the maximum number of instances supported in
 	// one instance group on GCP.
 	GCPMaxInstancesInInstanceGroup = 2000
+
+	// AffinityConfirmCount is the number of needed continuous requests to confirm that
+	// affinity is enabled.
+	AffinityConfirmCount = 20
```

Comment:
10 is probably enough? Assuming 3 backends, the possibility of getting a true negative is already lower than 0.01%...

Reply:
10 is fair enough.